package/MDAnalysis/analysis/nucleicacids.py

```diff
@@ -92,21 +108,32 @@ class NucPairDist(AnalysisBase):
 
     Attributes
     ----------
-    times: numpy.ndarray
-        Simulation times for analysis.
     results.pair_distances: numpy.ndarray
         2D array of pair distances. First dimension is simulation time, second
         dimension contains the pair distances for each each entry pair in
         selection1 and selection2.
 
         .. versionadded:: 2.4.0
 
+    results.distances: numpy.ndarray
+        stored in a 2d numpy array with first index selecting the Residue pair, 
+        and the second index selecting the frame number
```

Comment:
How is `distances` different from `pair_distances`?

Reply:
And if it's the same then you need to (sigh) deprecate the first one as it's part of the API.

Reply:
I think the change from `pair_distances` to `distances` was a suggestion from above, I added a deprecation notice in the docs and a todo in the code to remove it.